CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ option(ENABLE_SLIC "Build Umpire with SLIC logging" Off)
 option(ENABLE_STATISTICS "Track statistics for allocations and operations" Off)
 option(ENABLE_COVERAGE "Enable code coverage (with GCC)" Off)
 option(ENABLE_TOOLS "Enable Umpire Development Tools" On)
+option(ENABLE_PEDANTIC_WARNINGS "Enable Pedantic Compiler Warnings" On)
```

Comment:
👍

Reply:
Why is the description in title case? 

```suggestion
option(ENABLE_PEDANTIC_WARNINGS "Enable pedantic compiler warnings" On)
```

Reply:
I don't know.  I just copied from the previous line

Reply:
I think you wrote that line too 😄